### ISSUE TITLE
[Build] Dockerfile build fails due to legacy ius-release.rpm links

### ISSUE THREAD

**Issue by ilesser** [The-OpenROAD-Project/OpenROAD-flow#79]
**Describe the bug**
Following the *Option 2: Building the tools using docker* fails due to `https://centos7.iuscommunity.org/ius-release.rpm` link being broken.

As stated in [this iusrepo issue](https://github.com/iusrepo/announce/issues/18) some links will be decommissioned on 2020-06-01, in particular `https://centos7.iuscommunity.org/ius-release.rpm` which is used in many Dockerfiles.

**Expected behavior**
Dockerfile should build without errors.

**Screenshots**
Error after running `./build_openroad.sh`
![image](https://user-images.githubusercontent.com/16288620/83327540-03ebed00-a253-11ea-94d3-8ee542109987.png)

**Environment:**
Please only report issues for supported OSs.
 - OS: Linux Mint 19.1 Tessa [CentOS 7 in docker]
 - OpenROAD-flow commit [229a1f41]

**Comment by ilesser** [comment]
Changing the links to `https://repo.ius.io/ius-release-el7.rpm` should fix everything.

I can provide a PR during the weekend.

**Comment by ilesser** [comment]
Updating the links fixed the first error but now there is another error since it can't find `git2u`:
![image](https://user-images.githubusercontent.com/16288620/83328876-a8bef800-a25c-11ea-9758-3b05c1ece82d.png)

**Comment by zhanghb97** [comment]
> Updating the links fixed the first error but now there is another error since it can't find `git2u`:
> ![image](https://user-images.githubusercontent.com/16288620/83328876-a8bef800-a25c-11ea-9758-3b05c1ece82d.png)

You can find the available git version with `repoquery --whatprovides git`.

As for me, I can find the following versions:
```
repoquery --whatprovides git 
git-0:1.8.3.1-21.el7_7.x86_64 
git-0:1.8.3.1-22.el7_8.x86_64 
git222-0:2.22.2-1.el7.ius.x86_64 
git222-0:2.22.3-1.el7.ius.x86_64 
git224-0:2.24.2-1.el7.ius.x86_64
```

I solve the problem by using git224 (modify the Dockerfile `./tools/OpenROAD/Dockerfile`).

**Comment by rovinski** [comment]
Fixed in ed2ecee.
